core/docs/spark-qualification-tool.md

```diff
@@ -981,14 +981,14 @@ Sample output in text:
 Note: This step is optional.
 
 ```bash
-git clone https://github.com/NVIDIA/spark-rapids.git
-cd spark-rapids
-mvn -Pdefault -pl .,tools clean verify -DskipTests
+git clone https://github.com/NVIDIA/spark-rapids-tools.git
+cd spark-rapids-tools/core
+mvn -Pdefault clean verify -DskipTests
 ```
 
 The jar is generated in below directory :
 
-`./tools/target/rapids-4-spark-tools_2.12-<version>.jar`
+`./target/rapids-4-spark-tools_2.12-<version>.jar`
```

Comment:
I suggest to keep building instrictions in the core/index.md https://github.com/NVIDIA/spark-rapids-tools/tree/main/core#build

Reply:
OK, i'll just link to that page and remove the actual steps in the spark-qualification doc